dms/models/mixins_thumbnail.py

```diff
@@ -14,73 +14,82 @@ class Thumbnail(models.AbstractModel):
     _name = "dms.mixins.thumbnail"
     _description = "Thumbnail Mixin"
 
-    custom_thumbnail = fields.Binary(
-        string="Custom Thumbnail", attachment=False, prefetch=False,
-    )
+    custom_thumbnail = fields.Binary(string="Custom Thumbnail", attachment=False)
```

Comment:
Why not being `attachment=True`?

Reply:
This makes no sense, but it's how it was before, and since it's not the root of the performance problem, I decided to not touch it.